docs/src/userguide/debugging.md

```diff
@@ -109,3 +109,42 @@ Your code will run a lot faster with this option enabled.
 
 ### Troubleshooting
 If you encounter any issue when using the debugger, Please do let us know about it over at the [Julia VS Code](https://github.com/julia-vscode/julia-vscode) repository.
+
+
+## Settings to speed-up the debugger
+### `ALL_MODULES_EXCEPT_MAIN`
+In order to make the debugger run faster we want to minimize the number of packages/modules which are interpreted.
+Let's say you need to debug the code you wrote, but also the functionality provided by the package Statistics. 
+Click on the Debugger Default Compiled extension setting.
```

Comment:
Worth noting that there is also a bit of UI for this, maybe?

Reply:
True. However, ALL_MODULES_EXCEPT_MAIN is something to be set in the settings.json I think, before doing any debugging, I don't know other way to do it. 

If we use ALL_MODULES_EXCEPT_MAIN, no other symbols appear to be loaded in the "All" list such that the user can further add other modules from the UI.

Reply:
Ok, if you debug one more time, all desired symbols appears in the "All" list. But these are expected to appear as compiled instead of interpreted, right?
![image](https://github.com/julia-vscode/docs/assets/27725572/241f3bdb-065b-4225-be12-14862b76087e)

If you make one of the modules in the "All" list to be "compiled", the "All" list dissapears. Looks like a little bug here.
![image](https://github.com/julia-vscode/docs/assets/27725572/dda1b9be-53cd-4855-939a-28c6de630194)

Reply:
After a few more tests I noticed I can do the following: add ALL_MODULES_EXCEPT_MAIN from UI.
![image](https://github.com/julia-vscode/docs/assets/27725572/30ba4c79-e52a-46bb-a1ea-784d38354531)

But now If I want to remove the Statistics, I just cannot do it. One workaround was to add it as compiled (again?), and then remove it, but this just doesn't work from the UI:
![image](https://github.com/julia-vscode/docs/assets/27725572/76dd3712-4171-4844-a314-7b334f67ce92)

Result after removing Statistics from the UI, hoping it will become interpreted. Unfortunately, this did not work:
![image](https://github.com/julia-vscode/docs/assets/27725572/d29de4e9-c064-47eb-bf7b-8a11f4bd0d22)

Reply:
@pfitzseb while I have managed to add ALL_MODULES_EXCEPT_MAIN from the UI, I have found the UI option quite unreliable, see the above issues with the Statistics and the "All" list not expanding properly etc.
Additionally, you don't want to get the GLMakie in the interpreted mode, not even a single time. Currently, I think it's best to add ALL_MODULES_EXCEPT_MAIN directly in the settings.json such that expensive packages will get compiled from the very first time.

Here I see 2 options:
1. 
   a) Fix the behavior of "All" such that "Statistics" can be marked as interpreted from the UI.
   b) This PR will wait for the a) fix.

2. Accept the PR as it is and update this section after the identified fixes are done.

What do you think?

Reply:
I'll just merge the PR :) Thanks again!